crates/witness-generator/src/eest_generator.rs

```diff
@@ -54,27 +62,34 @@ impl ExecSpecTestBlocksAndWitnessBuilder {
 
     /// Builds the `ExecSpecTestBlocksAndWitnesses` instance.
     pub fn build(self) -> Result<ExecSpecTestBlocksAndWitnesses> {
+        let input_folder = self.input_folder;
         let tag = self.tag;
         let include = self.include.unwrap_or_default();
         let exclude = self.exclude.unwrap_or_default();
 
-        let mut cmd = Command::new("./scripts/download-and-extract-fixtures.sh");
-        if let Some(tag) = tag {
-            cmd.arg(tag);
-        }
-        let output = cmd.output()?;
+        let (directory_path, delete_eest_folder) = if let Some(input_folder) = input_folder {
```

Comment:
When using a local EEST folder, the `delete_eest_folder` controls how the Drop trait works, since we don't want to delete that one since wasn't created from the script.

Reply:
Yep, I'll do.